config/checkstyle/checkstyle.xml

```diff
@@ -96,6 +96,9 @@
         <module name="MissingSwitchDefault"/>
         <module name="FallThrough"/>
         <module name="UpperEll"/>
+        <module name="DeclarationOrder">
+            <property name="ignoreModifiers" value="true"/>
```

Comment:
I set this property to `true` to avoid another load of changes related to reordering class/instance fields.  I figure it will be easier to digest those in a separate PR.